organizational-structure-overview.md

```diff
@@ -12,14 +12,14 @@ The OpenSSF is comprised of instances of the following categories of official gr
 - The **Governing Board** (GB) oversees the budget and legal status/structure of the OpenSSF. Board meetings may or may not be open to the public.
 - **Committees** are formed by, and report to, the Board. Broadly speaking, Committees handle non-technical matters as needed by the GB, wherein a named set of people are tasked with handling a specific objective. Like the Board meetings, Committee meetings may or may not be open to the public.
 - The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
-  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
+  - TAC membership is composed of three representatives elected bi-annually by all active contributors (as defined by the TAC) and four representatives elected on alternating years by the Governing Board. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
```

Comment:
I think it is 4 by community and 3 by governing board. I am not sure on refresh frequency, so @brianbehlendorf should confirm.

Reply:
The text comes straight from the charter: https://charter.openssf.org/ so if this needs to be fixed it should be change there first (by the GB).

Reply:
Yes it also says 4 community and 3 GB and also frequency is annually for both
"four representatives elected -annually by all active contributors to
technical projects (as defined by the TAC) and three representatives appointed by the Governing
Board"

Reply:
+ @brianbehlendorf to confirm

Reply:
I agree with @inferno-chromium. This split was discussed in the context of the last TAC election and the GB agreed to the split of "3 elected by the GB / 4 elected by technical community". I believe the charter was then modified accordingly. Please see the notes in the TAC meeting minutes [1] (2021-12-14).

In general, we should avoid duplicating these statements across the charter and other documents for avoid inconsistencies.

[1] https://docs.google.com/document/d/18BJlokTeG5e5ARD1VFDl5bIP75OFPCtzf77lfadQ4f0/edit#heading=h.m6xa1m7mj5s7

Reply:
Ok, my  bad. I had an older version of the charter on my computer. To avoid any future discrepancies I removed it entirely per @gkunz suggestion.